TeachingRecordSystem/src/TeachingRecordSystem.Core/Events/Models/ProfessionalStatus.cs

```diff
@@ -13,7 +13,7 @@ public record ProfessionalStatus
     public required int? TrainingAgeSpecialismRangeTo { get; init; }
     public required string? TrainingCountryId { get; init; }
     public required Guid? TrainingProviderId { get; init; }
-    public required Guid? InductionExemptionReasonId { get; init; }
+    public bool? ExemptFromInduction { get; set; }
```

Comment:
Probably need a migration for the reporting DB for this change too

Reply:
Right - is that something I missed when I did the degree type stuff and you rectified here https://github.com/DFE-Digital/teaching-record-system/commit/1124c3105ce26dc9392ae18b6d3d6c3a7ee3ef00